rpmio/rpmkeyring.c

```diff
@@ -304,10 +319,16 @@ rpmRC rpmKeyringVerifySig(rpmKeyring keyring, pgpDigParams sig, DIGEST_CTX ctx)
 	    rpmlog(rc ? RPMLOG_ERR : RPMLOG_WARNING, "%s\n", lints);
 	    free(lints);
 	}
-    }
-
-    if (keyring)
+	if (keyptr)
+	    *keyptr = key;
+   }
```

Comment:
Indentation seems one off for the `}`

Reply:
It's off in the next line, too, just in the opposite direction. Fixed.